renamePaths.js

```diff
@@ -0,0 +1,30 @@
+
+if (app.documents.length > 0) {
+  var bkDoc = app.activeDocument;
+
+  // check for breakmark template
+  if (bkDoc.name == "breakmark-template.ai") {
+    var allArtboards = bkDoc.artboards;
+    app.selection = null;
```

Comment:
What is `app.selection`? we iterate on it below, but I don't see it getting set to anything?

Reply:
`bkDoc.selectObjectsOnActiveArtboard();` is an illustrator function that adds objects to `app.selection`